components/fxa-client/src/internal/state_persistence.rs

```diff
@@ -99,11 +99,12 @@ pub(crate) struct StateV2 {
     #[serde(default)]
     pub(crate) commands_data: HashMap<String, String>,
     #[serde(default)]
-    pub(crate) device_capabilities: HashSet<DeviceCapability>,
```

Comment:
I thought it would be okay since they're mostly just used as a cache for `ensure_capabilities`.  If we end up making an extra device update requests that seems okay.

However, the other use is in `reregister_current_capabilities()` and I realize that we really should not be using `server_local_device_info` at all for that.  In that scenario, we don't care what the server thinks our capabilities are, we want to re-send what we know our capabilities are.

I'm going to add back `device_capabilities` and have that track whatever was last sent to `ensure_capabilities` or `initialize_device`.  Eventually, I think we should track 2 things: the correct `LocalDevice` info that we know we have, and the `LocalDevice` info that the server thinks we have.